Hymns/Display/VerseView.swift

```diff
@@ -8,15 +8,15 @@ struct VerseLineView: View {
 
     var body: some View {
         HStack(alignment: .top) {
-            Text(viewModel.verseNumber ?? "").font(viewModel.fontSize.font).frame(minWidth: viewModel.fontSize.minWidth)
+            Text(viewModel.verseNumber ?? "").font(viewModel.fontSize.font).frame(minWidth: 12)
             VStack(alignment: .leading) {
                 if transliterate {
                     viewModel.transliteration.map { transliteration in
                         Text(transliteration).font(viewModel.fontSize.font)
                     }
                 }
                 Text(viewModel.verseText).font(viewModel.fontSize.font)
-            }.lineSpacing(5).padding(.bottom, 5)
+                }.fixedSize(horizontal: false, vertical: true).padding(.bottom, 5).lineSpacing(5)
```

Comment:
This deals with the issue of having lyrics do .... and us losing words on songs. This eliminates the issue entirely and just has it wrap to the next line.

Reply:
Without the animation, the "..." never appears anymore, so I think we can get rid of this

Reply:
It happens on animation but it also happens when a lines text does not fit. For instance try an iphone 8 emulation and the song "in this godless age."

Reply:
ah, gotcha. you are right!